app/views/prison/visits/show.html.erb

```diff
@@ -0,0 +1,31 @@
+<% content_for :header do %>
+  <%= t('.title') %>
+<% end %>
+
+<%= render(partial: 'people_box') %>
+
+<% unless @booking_response.processable? %>
+  <h2><%= t(".#{@booking_response.processing_state_name}") %></h2>
+<% else %>
```

Comment:
You're going to hate me for this, but one more thing: use `if`/`else`, not `unless`/`else`.